IndicoV2/Submissions/SubmissionsClient.cs

```diff
@@ -145,12 +145,22 @@ public async Task<ISubmission> GetAsync(int submissionId, CancellationToken canc
 
         public async Task<ISubmission> MarkSubmissionAsRetrieved(int submissionId, bool retrieved = true, CancellationToken cancellationToken = default)
         {
-            var resultId = await _strawberryShakeClient.Submissions().MarkRetrieved(submissionId, retrieved, cancellationToken);
-            var result = await _strawberryShakeClient.Submissions().List(new List<int?>(submissionId).AsReadOnly(), default, default, default, default, cancellationToken);
-            return new SubmissionSs(result?.Submissions?[0]);
+            await _strawberryShakeClient.Submissions().MarkRetrieved(submissionId, retrieved, cancellationToken);
+            var result = await _strawberryShakeClient.Submissions().Get(submissionId, cancellationToken);
+            return new Submission
```

Comment:
you should be able to use the `ToSubmissionFromSs` function below this one I believe

Reply:
The typing isn't working out for me due to the different return type between ListSubmissions and GetSubmission `IListSubmission_Submissions_Submissions` type versus `IGetSubmission_Submissions`